.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,81 @@
+---
+name: Bug Report
+about: Report an issue with openvas scanner
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+If reporting an issue please try to provide the information asked below.
+
+Before reporting an issue please:
+
+1. Be aware that this is not a support forum. If your issue is rather a question
+   than a bug report, please use our community forum at
+   https://community.greenbone.net/c/gse instead.
+2. Make sure that you're using the latest published GVM components for the
+   release you are using: https://community.greenbone.net/t/about-the-source-edition-gse-category/176
+3. Check the list of issues whether it isn't already reported.
+4. Read 1. again and if you still believe you found a software bug please
+   continue to file this issue. If you are in doubt use
+   https://community.greenbone.net/c/gse instead.
+
+Thanks for your help to keep the communication channels clean and consistent!
+-->
+
+### Expected behavior
+
+<!--
+  How did you expect openvas to behave?
+  Please write down how the web interface should work in your opinion.
+-->
+
+### Actual behavior
+
+<!--
+  Did something go wrong?
+  Is something broken, or not behaving as you expected?
+  Is this really an bug? If in doubt please use
+  https://community.greenbone.net/c/gse instead.
+  Please attach screenshots if possible! They are extremely helpful for
+  diagnosing issues.
+-->
+
+### Steps to reproduce
+
+<!--
+  How would you describe your issue to someone who doesn’t know openvas?
+  Try to write a sequence of steps that anybody can repeat to see the issue.
+-->
+
+1.
+2.
+3.
+
+### GVM versions
```

Comment:
Do we need to include ospd and ospd-openvas in this section as well?

Reply:
I don't know. @jjnicola ?

Reply:
@bjoernricks, yes, I think we need the ospd-openvas version.